.github/workflows/dvc-diff.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       run: |
         echo -e "## Summary of changed images\n" > report.md
         echo -e "This is an auto-generated report of images that have changed on the DVC remote\n" >> report.md
-        dvc diff --show-md master HEAD >> report.md
+        dvc diff --show-md main HEAD >> report.md
```

Comment:
This reminds me, we'll need to rename the 'master' branch on https://dagshub.com/GenericMappingTools/pygmt to 'main' too.